build_library/vm_image_util.sh

```diff
@@ -1224,7 +1224,7 @@ EOF
     "version": "${FLATCAR_VERSION_ID}",
     "providers": [{
       "name": "${provider}",
-      "url": "$(download_image_url "$(_dst_name ".box")")",
+      "url": "$(_dst_name ".box")",
```

Comment:
I can see that this change is basically a noop due to `download_image_url` being short-circuited when upload is disabled, thus printing just the passed parameter. This means that the `url` field was an invalid field for a long time and no one complained about it. We probably could think about using bincache or release server url, but I guess it is not important anyway, so whatever.

Reply:
You mean it's not really a URL? Yeah, I could probably put the bincache address here. I think it would have to be hardcoded though, none of the surrounding code is aware of that address. It's usually set by CI and only used for uploading at the end of each job.

Reply:
Yeah, maybe it could be something like `https://${BUILDCACHE_SERVER:-bincache.flatcar-linux.net}/images/${BOARD%-usr}/${FLATCAR_VERSION}`?